addons/website_slides_survey/models/slide_channel.py

```diff
@@ -8,3 +8,27 @@ class Channel(models.Model):
     _inherit = 'slide.channel'
 
     nbr_certification = fields.Integer("Number of Certifications", compute='_compute_slides_statistics', store=True)
+
+    def _remove_membership(self, partner_ids):
```

Comment:
Do we need to have those changes in survey?

Can't we leave the certification attempts untouched?

Not a fan of enabling the archiving mechanism on the survey user input just for a side e-learning case.

Reply:
@awa-odoo not sure about these changes so for now made a separate commit for this, will squash it once the review will be done.

Reply:
Maybe check with the PO, but indeed I would prefer to not change that much.

I think it's fair to keep certification attempts not archived even if you left the course.